extensions/jira/src/components/IssueActions.tsx

```diff
@@ -169,6 +169,15 @@ export default function IssueActions({
       </ActionPanel.Section>
 
       <ActionPanel.Section>
+        {issue.fields.parent && (
+          <Action.Push
+            target={<IssueDetail initialIssue={issue.fields.parent} issueKey={issue.fields.parent?.key} />}
+            title="Open Parent Issue"
+            icon={Icon.ChevronUp}
+            shortcut={{ modifiers: ["cmd", "shift"], key: "n" }}
```

Comment:
This shortcut is already used to add a new comment. Could you use another one? 🙂

Reply:
Of course. Which one do you think would be suitable?

Reply:
What about using the same shortcut as for `Open Child Issues`? I assume the actions should not be displayed at the same time as each issue either is a subtask and has a parent or is an issue and has subtasks.

Reply:
@thomaslombart The shortcut of the new command `Open Parent Issue` is now set to `cmd` + `shift` + `o` . It's the same shortcut as for the action `Open Child Issues`.

Reply:
It seems issues can have children and a parent at the same time. I changed the shortcut to `cmd` + `shift` + `u`.